packages/docs/src/routes/docs/cheat/components.mdx

```diff
@@ -17,15 +17,16 @@ export const Greeter = component$(() => {
 ## Props
 
 ```tsx
-import { component$, useStore } from '@builder.io/qwik';
+import { component$ } from '@builder.io/qwik';
 
 interface GreeterProps {
   salutation?: string;
   name?: string;
 }
-export const Greeter = component$((props: GreeterProps) => {
-  const salutation = props.salutation || 'Hello';
-  const name = props.name || 'World';
+
+export const Greeter = component$(({ salutation, name }: GreeterProps) => {
+  const salutation = salutation || 'Hello';
```

Comment:
@gilf wouldn't this type of assignment produce a `'salutation' has already been declared` error?

Reply:
Yep. Forgot to add default value in the destructing. Will change that and commit again.

Reply:
Thanks!